src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/ODataResourceSetSerializer.cs

```diff
@@ -235,6 +235,55 @@ public override void WriteObject(object graph, Type type, ODataMessageWriter mes
             return resourceSet;
         }
 
+        /// <summary>
+        /// Creates a function that takes in an object and generates nextlink uri.
+        /// </summary>
+        /// <param name="resourceSet">The resource set describing a collection of structured objects.</param>
+        /// <param name="resourceSetInstance">The instance representing the resourceSet being written.</param>
+        /// <param name="resourceSetType">The EDM type of the resourceSet being written.</param>
+        /// <param name="writeContext">The serializer context.</param>
+        /// <returns>The function that generates the NextLink from an object.</returns>
+        internal virtual Func<Object, Uri> GetNextLinkGenerator(ODataResourceSetBase resourceSet, IEnumerable resourceSetInstance, IEdmCollectionTypeReference resourceSetType,
```

Comment:
>internal virtual [](start = 8, length = 16)

internal virtual or public virtual?

Reply:
Changed it to just internal.

---
In reply to: [281402755](https://github.com/OData/WebApi/pull/1724#discussion_r281402755) [](ancestors = 281402755)